src/fiskaltrust.Launcher/ServiceInstallation/LinuxSystemD.cs

```diff
@@ -7,10 +7,14 @@ public class LinuxSystemD : ServiceInstaller
     {
         private static readonly string _servicePath = "/etc/systemd/system/";
         private readonly string _serviceName = "fiskaltrustLauncher";
-
-        public LinuxSystemD(string? serviceName, LauncherExecutablePath launcherExecutablePath) : base(launcherExecutablePath)
+        private readonly string _serviceUser;
+        private readonly string _requiredDirectory = "/var/lib/fiskaltrust";
+        
+        public LinuxSystemD(string? serviceName, LauncherExecutablePath launcherExecutablePath)
+            : base(launcherExecutablePath)
         {
-            _serviceName = serviceName ?? _serviceName;
+            _serviceName = serviceName ?? "fiskaltrustLauncher";
+            _serviceUser = Environment.GetEnvironmentVariable("USER") ?? "defaultUser";
```

Comment:
Is `"defaultUser"` a real linux thing? If not that would potentialy make things worse to change the owner to some user that does not exist. I that case we should just do nothing.

Reply:
I wanted to have possibility to change to correct username and add instruction in the READme. 
However, I can remove this and add some Log.Warning